examples/wave-eager.py

```diff
@@ -94,7 +96,7 @@ def rhs(t, w):
         fields = rk4_step(fields, t, dt, rhs)
 
         if istep % 10 == 0:
-            print(istep, t, la.norm(fields[0].get()))
+            print(istep, t)
```

Comment:
You could call `discr.norm(fields[0], np.inf)` here.

Reply:
Fixed, thanks.